# estimate
Header header

int8 receiver_id
int8 estimation_done
int8 initialized

float32[] anchor_offset

int32[] pose_id

float32[] estimate